lms/templates/dashboard/_dashboard_course_listing.html

```diff
@@ -76,7 +76,7 @@
         % elif verification_status.get('status') == VERIFY_STATUS_APPROVED:
           <span class="sts-enrollment" title="${_("You're enrolled as a verified student")}">
             <span class="label">${_("Enrolled as: ")}</span>
-            <img class="deco-graphic" src="${static.url('images/verified-ribbon.png')}" alt="${_("ID Verified Ribbon/Badge")}" />
+            <img class="deco-graphic" src="${static.url('images/verified-ribbon.png')}" alt="" />
```

Comment:
@marcotuts here as you can observe alt message has been removed so there is no need for testing alt message.